packages/waku/package.json

```diff
@@ -65,7 +65,8 @@
     "./router/server": {
       "types": "./dist/router/server.d.ts",
       "default": "./dist/router/server.js"
-    }
+    },
+    "./vite-rsc/*": "./dist/vite-rsc/*.js"
```

Comment:
Is it necessary?

Reply:
Yes, this should be limited further and technically they might not be necessary at all (or at least now).
My thinking is that, Waku can work purely as a normal plugin (both dev and build) and users can use it inside own `vite.config.ts`. Also exporting base server entry `entry.rsc.js` can be helpful later to allow letting users write own server entry e.g. for cloudflare `export default { fetch, etc... }`. (As an example, see `examples/01_template_cf/src/worker.tsx` in  https://github.com/hi-ogawa/waku/pull/15/ though this is not currently working at all).

I think expanding the API or features should be done later, so for now, I'll try to hide these exports.

Reply:
> for now, I'll try to hide these exports.

That sounds good.

Reply:
> Waku can work purely as a normal plugin (both dev and build) and users can use it inside own vite.config.ts

By the way, I don't dislike the idea. It's actually our secondary goal.